asylum-connect/catalog-api
# Feature/1199612928739222/last updated filter

## Description

Add filter by last updated date in control panel

- Asana ticket: https://app.asana.com/0/1132189118126148/1199612928739222

## PR Checklist



- [ X] Assign @FJKhan **and** @Alfredo-Moreira as reviewers.
- [X ] If your PR is not a hotfix, is it targeted for `dev`?

## How to Test

1. Login to Control Panel.
2. Under Filter Organizations, enter a date or use the date picker to populate the Updated Before: field
3. Click Search
4. Click View next to one of the returned results to view the organization
5. Verify that the Last Verified date is on or before the date selected.
